.gitignore

```diff
@@ -5,3 +5,7 @@ Cargo.lock
 *.iml
 .idea/
 .vscode
+
+# Serialized generated prover & verifier state used by plonky2
+prover_state_*
+verifier_state_*
```

Comment:
This is a leftover I think. We now have circuits written into the `circuits/` folder.